src/Core/ProductInfo.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Reflection;
+
+namespace Azure.DataApiBuilder.Core;
+
+public class ProductInfo
```

Comment:
```suggestion
public static class ProductInfo
```

Should probably make this class static as we don't need it to be instantiaed anywhere.